packages/form/src/SelectRow.js

```diff
@@ -50,6 +50,22 @@ function getValueLabelMap(fromChildren = []) {
     return resultMap;
 }
 
+/**
+ * Generate a value-avatar map from all `<SelectOption>`s.
+ *
+ * @param {array} fromOptions
+ * @return {Map}
+ */
+function getValueAvatarMap(fromChildren = []) {
```

Comment:
Can this be merged with `getValueLabelMap` so we don't need to iterate the Array twice, only for deriving different data?

Reply:
Nice suggestion. Thanks!